packages/api/src/schema/schema.yaml

```diff
@@ -239,6 +241,45 @@ components:
           example: D8321C3E-B29C-45EB-A1BB-A623D8BE0F65
           description:
             ID of object store where to which this stream was recorded
+        pushTargets:
+          type: array
+          example: [ { "id" :"PUSH123", "profile": "720p" } ]
+          description:
+            References to targets where this stream will be simulatenously pushed to
+          items:
+            type: object
+            required: [profile]
+            additionalProperties: false
+            properties:
+              profile:
+                type: string
+                description:
+                  Name of transcoding profile that should be sent. Use "source"
+                  for pushing source stream data
+                minLength: 1
+                maxLength: 500
+                example: aac
+              id:
+                type: string
+                description: ID of push target object where to push this stream
+              inline:
```

Comment:
I like having this in a separate object but I think maybe it should be named `pushTarget`

Reply:
Don't you think it might be weird to have the `pushTargets{}.pushTarget` repetition?

Later yesterday I thought of also updating the `id` field to make it explicit on the field what it is the ID for. So I though of the combination `pushTarget{}.targetId` OR `pushTarget{}.targetSpec`, borrowing the `spec` from k8s objects. What do you think?

Reply:
FTR: decided to go with `id` and `spec` as a naming pattern that we expect to follow on these "reference or inline object" API objects.